src/index.md

```diff
@@ -1298,6 +1298,14 @@ The fix for the main issue should remove the test from the ProblemList or remove
 
 After a failure is handled by excluding a test, the main JBS issue should be re-triaged and possibly given a new priority. This should be handled by the standard triage process. A test exclusion results in an outage in our testing. This outage should be taken into consideration when triaging, in addition to the impact of the bug itself.
 
+## GitHub Actions
+
+GitHub has a feature called **GitHub Actions** (GHA) that can be used to automate testing. The GHA is executed whenever a PR is created or updated in the repository. The GHA in the JDK project is configured to run a set of tests that is commonly known as **tier 1**. This is a relatively fast, small set of tests that tries to verify that your change didn't break the JDK completely. In tier 1 the JDK is built on a small set of platforms including (but not necessarily limited to) Linux, Windows, and MacOS, and a few tests are executed using these builds.
```

Comment:
One correction: Once enabled in your repo it runs on a push to a branch in your repo, not just when you create a PR. Skara will show the results of the test run when you create or update a PR.

Reply:
Thanks for this clarification!